CenterForOpenScience/osf.io
# [PLAT-851] Storage internationalization frontend

## Purpose

This allows the user to interact with their storage regions in various ways.

## Changes

- show storage location on node settings page
<img width="1435" alt="screen shot 2018-05-18 at 4 07 26 pm" src="https://user-images.githubusercontent.com/9688518/40255558-9856bf0c-5ab5-11e8-8e59-22c530abf26f.png">
- Have fangorn show region names on provider listings
<img width="1429" alt="screen shot 2018-05-18 at 4 09 11 pm" src="https://user-images.githubusercontent.com/9688518/40255632-d5d0bb1c-5ab5-11e8-999f-da8dd4406597.png">
- allow user to set storage location on node creation 
<img width="1438" alt="screen shot 2018-05-18 at 4 10 50 pm" src="https://user-images.githubusercontent.com/9688518/40255696-1164cc90-5ab6-11e8-9125-a2861411f607.png">
- allow user to change thier default node location

![user_profile_storage_location](https://user-images.githubusercontent.com/9688518/40256251-e3830380-5ab7-11e8-9451-05b101e7ba47.gif)

- Update welcome email



## QA Notes

- check welcome email text is correct.
- check fangorn file tree storage info is correct
- check/change the default user storage location
- check/change the default when storage location when creating a node
- check the node setting page to confirm the storage location.

## Documentation

Should be covered in the osf.help docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-851
